### PR TITLE
Hundred multiplier to Plotly text format

### LINKED ISSUE #64: format plotly template strings with functions
doesn't need to be in this pr but these string pieces could be simplified with a function

_Originally posted by @MaxGhenis in https://github.com/UBICenter/us-calc/pull/62#discussion_r628204790_

### LINKED ISSUE #66: Remove multiplications by 100
Currently causing an off-by-100 error, and confusing. Instead keep everything as 0-1 proportions, and use `fig.update_layout(yaxis_tickformat="%")` instead of `suffix="%"` to show proportions as proper percentages.

### PR DESCRIPTION
**Pull request by fedderw**
fixes #66 - too many instances of multiple decimals by `* 100` to get values in  % terns,
fixes #64 - "Include in" instead of "exclude from" ubi
@MaxGhenis

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -835,39 +818,35 @@ def pv_rate(column):
 
     fig.update_yaxes(
         # title_text = "Percent change",
-        ticksuffix="%",
+        # ticksuffix="%",
```
Comment (MaxGhenis) [inline app.py]: Can this be removed? It'll be `tickformat="%"` right?

File: app.py [hunk 2]
```diff
@@ -796,13 +772,17 @@ def pv_rate(column):
         "Black",
         "Hispanic",
     ]
-
+    fig_COLS=[
```
Comment (MaxGhenis) [inline app.py]:
```suggestion
    fig_cols=[
```
(uppercase indicates constants)

File: app.py [hunk 3]
```diff
@@ -681,21 +680,18 @@ def return_all_state(metric):
...
 
     # Calculate the change in poverty rate
     target_persons["poor"] = target_persons.new_resources < target_persons.spmthresh
     total_poor = (target_persons.poor * target_persons.asecwt).sum()
-    poverty_rate = (total_poor / population) * 100
-    poverty_rate_change = (
-        (poverty_rate - original_poverty_rate) / original_poverty_rate * 100
-    ).round(1)
-
+    # TODO fix thism, the default  poverty rate is too high
```
Comment (MaxGhenis) [inline app.py:686]: Is this still the case?

**Review by MaxGhenis — COMMENTED** [review]
Nice, this looks a lot cleaner. Aside from these questions, could you also run `black`?

**Comment by MaxGhenis** [comment]
Thanks @fedderw, is this good to merge now?

**Comment by fedderw** [comment]
Merge away

**Pull request merged**